content/en/_index.html

```diff
@@ -3,26 +3,138 @@
 linkTitle = "cdevents.dev Homepage"
 
 +++
-
+<!-- cSpell:locale en-US -->
 {{< blocks/cover title="CDEvents" subtitle="A common specification for Continuous Delivery events" image_anchor="top" color="primary" height="min">}}
 <div class="mx-auto">
 	<a class="btn btn-lg btn-dark mr-3 mb-4" href="docs">
-		Get Started <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+		Read the Docs <i class="fas fa-arrow-alt-circle-right ml-2"></i>
 	</a>
 	<a class="btn btn-lg btn-light mr-3 mb-4" href="https://github.com/cdevents">
 		GitHub <i class="fab fa-github ml-2 "></i>
 	</a>
 </div>
+<div>
+	Embrace the freedom to pick and choose as you will, empower your builds with broad reusability and free your Continuous Delivery approach from vendor lock-in!
+<br/><br/>
+</div>
+<div>
+	With CDEvents, you have the freedom to adopt the Continuous Delivery approach that your product needs, with the flexibility to incorporate whatever tooling best suits your specific requirements.
+	<br/><br/>
+</div>
+<div>
+	CDEvents is a common specification for Continuous Delivery events, enabling interoperability in the complete software production ecosystem.
+	<br/>
+</div>
```

Comment:
I think the text in the first section of the homepage shouldn't be more than a sentence.
Or perhaps more text could be included in a smaller font.
Some examples I'm looking at are https://www.sigstore.dev/, https://tekton.dev/ and https://knative.dev/docs

Reply:
We could certainly do that, but it wouldn't be my recommendation in this case.

I believe that CDEvents is facing a different problem from those products, so it's perhaps inadvisable to copy their marketing approach because of that. Products like Tekton and Knative are well established, existing products that are well understood in their markets. CDEvents is a hypothesis about a product that would create a new market. Because of that, we need to pitch the product to the audience in a way that helps them to understand what it would do for them. The driver at this stage of maturity is to sell the dream, with words, until we have implementations to demonstrate with results...

Reply:
I think the text in this section is quite good, and I think it's ok to have 3 sentences here for now. I'd prefer to remove the exclamation mark and full stops though.